test/lsp.py

```diff
@@ -948,6 +948,11 @@ def setup_lsp(
                 params['initializationOptions'] = {}
             params['initializationOptions']['include-paths'] = custom_include_paths
 
+        if self.trace_io:
```

Comment:
Shouldn't the type of `trace_io` be declared along with other class members at the top of the class definition?